learning-resources/misc/index.md

```diff
@@ -66,6 +66,10 @@ What other resources can you find? Try out,
    - [Muhammad Kamran Qadri](https://web.facebook.com/groups/softdevpk/permalink/966099857180373/)
    - [Muhammad Kamran Qadri](https://web.facebook.com/groups/softdevpk/permalink/966755790448113/)
 
+3. [**Click here**](https://github.com/HypertextAssassin0273/Data_Structures_in_Cpp) to get custom & easy implementation of some **STL containers**, alongwith complete **FAST_Course_Spring_21** material (provided by [Shahzaib Khan](https://hypertextassassin0273.github.io/)).
```

Comment:
Since it's a repository, it would be a great idea if we could make a section for `Repository` entirely.

So this can be a pretty concise description as well, such as,

```markdown
## Repository

- [Custom C++ STL Containers Implementation](https://github.com/HypertextAssassin0273/Data_Structures_in_Cpp)
```

A direct link to "FAST_Course_Spring_21" can be provided - "provided by Shahzaib Khan" is redundant is it leads to the GitHub page anyways.